lib/ts/station/station.ex

```diff
@@ -4,9 +4,10 @@ defmodule Station do
 	Module that implements the interface of the Station.
 	"""
 
-	@behaviour TS.StationBehaviour
+	@behaviour Station.Behaviour
```

Comment:
please remove the dot.

Reply:
Station.Behaviour is now Station.StationBehaviour.